src/game/level.cpp

```diff
@@ -10,6 +10,185 @@ void Level::load_json(std::vector<std::shared_ptr<Entity>> *entities, const std:
   simdjson::padded_string json = simdjson::padded_string::load(real_path);
 
   this->json_data = this->json_parser.iterate(json);
+
+  simdjson::ondemand::object json_data_obj(this->json_data.get_object());
+  u32 json_data_size = json_data_obj.count_fields();
+  std::string layer_id;
+
+  for (u32 i = 0; i < (json_data_size - 14); i++) {
+    if (i == 0)
+      layer_id = "/layers";
+    else
+      layer_id = "/layers" + std::to_string(i);
+
+    simdjson::ondemand::array layers = this->json_data.at_pointer(layer_id).get_array();
+
+    simdjson::ondemand::array background_array = layers.at_pointer("/0/chunks").get_array();
+
+    u32 bg_array_size = background_array.count_elements();
+
+    for (u32 i = 0; i < bg_array_size; i++) {
+      std::string json_pointer = layer_id + "/0/chunks/" + std::to_string(i);
+
+      simdjson::ondemand::object obj(this->json_data.at_pointer(json_pointer).get_object());
+
+      f64 x = obj["x"];
+      f64 y = obj["y"];
+
+      std::vector<u32> values;
+
+      simdjson::ondemand::array arr((obj["data"].get_array()));
+
+      for (f64 value : arr)
+        values.push_back(value);
+
+      this->background_content.emplace_back(x, y, std::move(values));
```

Comment:
`this->background_content.emplace_back(obj["x"], obj["y"], std::move(values))`

note: i have no idea if simdjson would actually allow this